pallene/builtins.lua

```diff
@@ -4,6 +4,7 @@ local builtins = {}
 
 for lua_name, typ in pairs({
     ["io.write"]     = types.T.Function({types.T.String()}, {}),
+    ["sqrt"]         = types.T.Function({types.T.Float()}, {types.T.Float()}),
```

Comment:
Could we rename it to `math.sqrt`?

Reply:
Good idea. Is more consistent with io.write.